pkgs/tools/nix/nixos-install-tools/default.nix

```diff
@@ -18,9 +20,8 @@ in
   name = "nixos-install-tools-${version}";
   paths = lib.attrValues {
     # See nixos/modules/installer/tools/tools.nix
-    inherit (config.system.build)
-      nixos-install nixos-generate-config nixos-enter;
-
+    inherit (config.system.build) nixos-generate-config;
+    inherit nixos-install nixos-enter;
     inherit (config.system.build.manual) nixos-configuration-reference-manpage;
```

Comment:
Do we actually need this package? Than we could get rid of the nixos evaluation all together...

Reply:
It gets mentioned here https://github.com/NixOS/nixpkgs/blob/21576aba43e604a95cc45c25248f39f3989ebe36/nixos/doc/manual/contributing-to-this-manual.chapter.md#contributing-to-the-configurationnix-options-documentation-sec-contributing-options and is used by hydra https://github.com/NixOS/nixpkgs/blob/21576aba43e604a95cc45c25248f39f3989ebe36/nixos/release.nix#L149

Reply:
Yes. But release.nix is importing nixos directly instead of using pkgs. So this wouldn't be a blocker.